.circleci/config.yml

```diff
@@ -677,12 +677,11 @@ workflows:
       # Only runs on vX.X.X tags if all tests are green
       - publish_npm_package:
           filters:
-            branches:
```

Comment:
Prior to removing this block, we'd end up with two separate `publish_npm_package` jobs.

Removing it ensures we only run that job on commits that are tagged with a string that matches /v[0-9]+(\.[0-9]+)*(\-rc(\.[0-9]+)?)?/ (from the `tags:` block).